src/API/REST/V3/Routes/Subscriptions/Actions/GetSubscriptionItemSchema.php

```diff
@@ -31,6 +31,11 @@ public function __invoke(): array
                     'default' => 'live',
                     'enum' => ['live', 'test'],
                 ],
+                'campaignId' => [
+                    'type' => 'integer',
+                    'description' => esc_html__('Campaign ID', 'give'),
+                    'nullable' => true,
```

Comment:
Interesting. I didn't know that you could set `nullable` arg. Usually, you would do something like `['integer', 'null']`, but this looks nice 👍